### PR TITLE
[RQ-1202]: update workspace creation for appsumo flow

### PR DESCRIPTION
**Pull request by rohanmathur91**
Closes issue: 

## ✅ Checklist:

- [ ] Make sure linting and unit tests pass.
- [ ] No install/build warnings introduced.
- [ ] Verified UI in browser.
- [ ] For UI changes, added/updated analytics events (if applicable).
- [ ] For changes in extension's code, manually tested in Chrome and Firefox.
- [ ] For changes in extension's code, both MV2 and MV3 are covered.
- [ ] Added/updated unit tests for this change.
- [ ] Raised pull request to update corresponding documentation (if already exists).

### CODE REVIEW

File: app/src/components/landing/Appsumo/Appsumo.tsx [hunk 1]
```diff
@@ -124,53 +115,81 @@ const AppSumoModal: React.FC = () => {
...
 
+  const createNewWorkspaceForAppSumo = useCallback(async () => {
+    setIsLoading(true);
+
+    const newTeamName = "Team Workspace";
+    const createTeam = httpsCallable(getFunctions(), "teams-createTeam");
+    try {
+      const response: any = await createTeam({ teamName: newTeamName });
+      trackNewTeamCreateSuccess(response?.data?.teamId, newTeamName, "appsumo");
+      console.log("1. workspace to be upgraded :: ", response?.data?.teamId);
```
Comment (nafees87n) [inline app/src/components/landing/Appsumo/Appsumo.tsx:126]: log missed

File: app/src/components/landing/Appsumo/Appsumo.tsx [hunk 2]
```diff
@@ -124,53 +115,81 @@ const AppSumoModal: React.FC = () => {
     await batch.commit();
   }, [db, appsumoCodes]);
 
+  const createNewWorkspaceForAppSumo = useCallback(async () => {
+    setIsLoading(true);
+
+    const newTeamName = "Team Workspace";
```
Comment (nafees87n) [inline app/src/components/landing/Appsumo/Appsumo.tsx:121]: Are we keeping the new team's name as `Team Workspace`? In the create team flow, we give default name as `<user>'s Workspace (<domain>)`

**Review by nafees87n — APPROVED** [review]
Looks good

File: app/src/components/landing/Appsumo/Appsumo.tsx (hunk 2 above)
Comment (rohanmathur91) [reply app/src/components/landing/Appsumo/Appsumo.tsx:121]: For appsumo we are keeping `Team Workspace` as default name

**Pull request merged**